### PR TITLE
Increase the space below headings

### LINKED ISSUE #50: Update the Bootstrap theme
Since we are using Bootstrap with SCSS, we can change the default colours, spacing and so on. At minimum we should increase the space below headings, since it looks like there isn't much space after them right now.

https://getbootstrap.com/docs/4.4/getting-started/theming/

### PR DESCRIPTION
**Pull request by abdulkadiret**
Increased the space below heading a little bit

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
+// Your variable overrides
+$headings-margin-bottom: 1rem;
+
+// Bootstrap and its default variables
+@import '../node_modules/bootstrap/scss/bootstrap';
```
Comment (bonboh) [inline src/App.scss]: We are already importing Bootstrap in index.scss so we don't want to import it again

File: src/App.scss [hunk 2]
```diff
@@ -1,3 +1,9 @@
+// Your variable overrides
+$headings-margin-bottom: 1rem;
```
Comment (bonboh) [inline src/App.scss]: The override should go in index.scss which is the top level SCSS file. App.scss is meant to be the styling for App.scss (which also means `footer-link` inside App.scss probably should be somewhere else, but we can leave that for now)

**Review by bonboh — CHANGES_REQUESTED** [review]
Good job finding the header spacing override. Just a couple of comments

File: src/App.scss (hunk 2 above)
Comment (abdulkadiret) [reply src/App.scss]: Moved!

**Review by bonboh — APPROVED** [review]
Good job. The whole team can decide on the default colours as well at some point

**Pull request merged**